raft/server.cc

```diff
@@ -578,7 +589,7 @@ future<> server_impl::wait_for_entry(entry_id eid, wait_type type, seastar::abor
     check_not_aborted();
 
     if (as && as->abort_requested()) {
-        throw request_aborted();
+        throw request_aborted(format("Abort requested while waiting for entry with idx: {}, term: {}", eid.term, eid.idx));
```

Comment:
`idx: {}` is given `eid.term` and `term: {}` is given `eid.idx`, should be reversed.